ExtraDry/ExtraDry.Blazor/Models/CommandInfo.cs

```diff
@@ -82,9 +80,6 @@ public CommandInfo(object viewModel, Func<Task> action)
     /// </summary>
     public string? Affordance { get; set; }
 
-    /// <inheritdoc cref="CommandAttribute.PropertyName" />
-    public string? PropertyName { get; set; }
```

Comment:
Be aware that this is a breaking a change and will likely tank FMI on import.  Not a big deal since you're doing the importing.  Often I just mark these as Obsolete for a release or two in case anyone else updates ED.